app/_src/production/install-kumactl.md

```diff
@@ -56,7 +56,7 @@ and extract the archive with `tar -xvzf {{ site.mesh_helm_install_name }}-{{ pag
 
 Add the `kumactl` executable to your path:
 ```
-cd kuma-{{ page.version_data.version }}/bin
+cd {{site.mesh_product_name_lowercase}}-{{ page.version_data.version }}/bin
```

Comment:
does https://shopify.github.io/liquid/filters/downcase/ not work?

Reply:
it works but we're going with `mesh_install_archive_name`